REQUIREMENTS.md

```diff
@@ -1,27 +1,24 @@
-# IPLD Requirements
+# IPLD Cornerstones
 
 This document outlines parts of IPLD that should and should not be changed to
-ensure the success of future improvements (especially type systems).
+ensure the success of future improvements and the continuity of direction.
 
 **Definitions**
 
 * Block: A block is a chunk of an IPLD DAG, encoded in a format. Blocks have CIDs.
-* Fragment: A piece of an IPLD DAG. Blocks contain fragments.
 * Node: A node is a *point* in an IPLD DAG (object, array, number, etc.).
-* Link: A link is an IPLD Node that points to another IPLD Node.
-* Path: A paths a human readable pointer to an IPLD Node.
+  Many nodes can exist encoded inside one Block.
+* Link: A link is a kind of IPLD Node that points to another IPLD Node.
+* Path: A path is composed of segments which each specify a step across an IPLD Node.
```

Comment:
I am struggling to grok `a step across an IPLD Node.` How does one step across a _point_? Would it be more accurate to say `step between IPLD Node**s**?

Reply:
Actually, I think it just clicked.
A Node is:
```
(A)
```
A Link is:
```
(A) -> (B)
      ^
    Link
```
A Path of `/A/B/C` represents:
```
(A) -> (B) -> (C)
  |__________^
           ^
(B) gets stepped over 
```
So maybe `step over IPLD Node(s)` is a better way of saying this. @warpfork?

Reply:
Yeah, I agree, this bit of text struggled.  "between" might do better.

I'm not sure about that diagram, because it's not necessary that it be a link in-between; rememeber that a whole tree of Nodes can be in a Block.

Maybe I should also break this down into bullet points for "Path" and "Path Segment", and that might help?  E.g. one Path Segment moves from one Node to the next (but only can describe movement of a single stride: e.g. looking up one map key, or stepping into an list by index).  A Path is just a collection of Path Segments, which we often use to describe a walk of a specific, er, well, *path* down a Node tree.

Reply:
Made another shot at this in: #152 -- does that one seems clearer to you?

Reply:
@warpfork Much better, made a note on a slight clarification.